cpp/src/unary/cast_ops.cu

```diff
@@ -470,4 +484,10 @@ std::unique_ptr<column> cast(column_view const& input,
   return detail::cast(input, type, stream, mr);
 }
 
+bool is_supported_cast(data_type from, data_type to) noexcept
+{
+  CUDF_FUNC_RANGE();
+  return detail::is_supported_cast(from, to);
```

Comment:
FWIW, I wonder if this is one of the cases where we don't want the detail/public split with a `CUDF_FUNC_RANGE` in the public API. This is presumably a cheap enough operation that tracking nvtx for it makes no sense.
```suggestion
  return double_type_dispatcher(from, to, detail::is_supported_cast_impl{});
```

Reply:
I agree with not including the `CUDF_FUNC_RANGE()` here.
There is no device code execution and it will just add noise to the profile output.
We can always add it back in if needed. Probably worth a comment in the code at least.

Reply:
Did that.